### PR TITLE
Capture and test against stdout

### PR DESCRIPTION
**Pull request by marek2901**

### CODE REVIEW

File: hoster/activate_test.go [hunk 1]
```diff
@@ -9,6 +10,7 @@ func TestActivationHandler_Handle(t *testing.T) {
 	subject := hoster.NewActivationHandler()
 
 	t.Run("Smoke test", func(t *testing.T) {
-		subject.Handle(fakeCliContext(t, []string{"test"}))
+		output := captureOutput(t, func() { subject.Handle(fakeCliContext(t, []string{"test"})) })
+		assert.Equal(t, "TODO activate test\n", output)
```
Comment (marcingol1) [inline hoster/activate_test.go:14]: 😂

File: hoster/common_test.go [hunk 2]
```diff
@@ -16,3 +19,17 @@ func fakeCliContext(t *testing.T, args []string) *cli.Context {
...
+	rescueStdout := os.Stdout
+	r, w, err := os.Pipe()
+	assert.NoError(t, err)
+	os.Stdout = w
+	f()
+	err = w.Close()
+	assert.NoError(t, err)
+	out, err := ioutil.ReadAll(r)
+	assert.NoError(t, err)
+	os.Stdout = rescueStdout
```
Comment (marcingol1) [inline hoster/common_test.go:33]: Nice, `captureOutput` seem to be quite verbose and easy to read

**Review by marcingol1 — APPROVED** [review]

File: hoster/common_test.go (hunk 2 above)
Comment (marek2901) [reply hoster/common_test.go:33]: Yes. Go is verbose. Sometimes too much, but it is fun how different it is :).

File: hoster/activate_test.go (hunk 1 above)
Comment (marek2901) [reply hoster/activate_test.go:14]: 🤷‍♂

**Pull request merged**